spec/fibered_mysql2_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+RSpec.describe FiberedMysql2 do
+  it "has a version number" do
+    expect(FiberedMysql2::VERSION).not_to be nil
+  end
```

Comment:
Lets start rspecifying some of these tests
```suggestion
  describe 'VERSION' do
    subject { FiberedMysql2::VERSION }
    it { should_not be_nil }
  end
```